webtransport/connect.any.js

```diff
@@ -1,6 +1,7 @@
 // META: global=window,worker
 // META: script=/common/get-host-info.sub.js
 // META: script=resources/webtransport-test-helpers.sub.js
+// META: script=/common/utils.js
```

Comment:
Why do we need this change?

Reply:
Oh sorry we'll need it to use token() but this is redundant. Removed.